src/adapter/src/coord/sequencer.rs

```diff
@@ -587,6 +587,8 @@ impl Coordinator {
                         }
                     };
 
+                    self.maybe_create_linked_cluster(source_id).await;
```

Comment:
Do we not need this for `sequence_create_sink`?

Reply:
Sinks are sequenced in two phases. So this happens for sinks in `create_storage_export` in ddl.rs.